.env.example

```
@@ -1,25 +1,63 @@
 PRIVY_SECRET = ''
-APP_MNEMONIC = 'candy maple cake sugar honey ... potato blue'
+APP_MNEMONIC = ''
 NEYNAR_API_KEY = ''
 SUPABASE_SERVICE_KEY = ''
 YOUTUBE_API_KEY = ''
+
 # These are specific to the environment
 # and so need to be configured locally
 # but are not sensitive data
-NEXT_PUBLIC_APP_FID = '123'
+NEXT_PUBLIC_APP_FID = ''
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)